vms/platformvm/txs/mempool/mempool.go

```diff
@@ -297,3 +304,30 @@ func (m *mempool) deregister(tx *txs.Tx) {
 	inputs := tx.Unsigned.InputIDs()
 	m.consumedUTXOs.Difference(inputs)
 }
+
+func (m *mempool) DropExpiredStakerTxs(minStartTime time.Time) []ids.ID {
```

Comment:
Should we keep this as a standalone function? We only seem to use exported `*mempool` functions - so I think we could keep this as a function rather than a method.

Reply:
Good call, done.